src/app/app.component.html

```diff
@@ -0,0 +1,40 @@
+<nav>
```

Comment:
@hitpopdimestop Then we should have another file renamed to this pattern. Anyway there is a file to rename and personally I would prefer to have `template.html` because it shows that it is actually template and nothing more